pkg/endpoint/restore.go

```diff
@@ -110,8 +110,8 @@ func ReadEPsFromDirNames(owner regeneration.Owner, basePath string, eptsDirNames
 // * regenerates the endpoint
 // Returns an error if any operation fails while trying to perform the above
 // operations.
-func (e *Endpoint) RegenerateAfterRestore() error {
-	if err := e.restoreIdentity(); err != nil {
+func (e *Endpoint) RegenerateAfterRestore(identityAllocator cache.IdentityAllocator) error {
```

Comment:
This seems to be a leftover since we have the allocator present in the endpoint.